ydb/core/tx/schemeshard/schemeshard_impl.h

```diff
@@ -1016,8 +1016,12 @@ class TSchemeShard
 
     void DescribeTable(const TTableInfo::TPtr tableInfo, const NScheme::TTypeRegistry* typeRegistry,
                        bool fillConfig, bool fillBoundaries, NKikimrSchemeOp::TTableDescription* entry) const;
-    void DescribeTableIndex(const TPathId& pathId, const TString& name, NKikimrSchemeOp::TIndexDescription& entry);
-    void DescribeTableIndex(const TPathId& pathId, const TString& name, TTableIndexInfo::TPtr indexInfo, NKikimrSchemeOp::TIndexDescription& entry);
+    void DescribeTableIndex(const TPathId& pathId, const TString& name,
+        bool fillConfig, bool fillBoundaries, NKikimrSchemeOp::TIndexDescription& entry
```

Comment:
I have added explicit `fillConfig` and `fillBoundaries` options to be able to control the behavior of `DescribeTableIndex`. Moreover, these options are supported by `DescribeTable` and it would be inconsistent to ignore them in `DescribeTableIndex`